src/labels/CustomLabels.labels

```diff
@@ -1959,6 +1967,14 @@
         <shortDescription>The RD2 Batch job can only be executed when Enhanced RD is enabled</shortDescription>
         <value>You must enable Enhanced Recurring Donations before you can run the Enhanced Recurring Donations batch job.</value>
     </labels>
+    <labels>
+        <fullName>RD2_HaveAlreadyBeenEnabled</fullName>
```

Comment:
Suggestion: `RD2_EnabledErrorMessage`?

@sam-knox , can you please verify the custom label?

Reply:
I have updated the label name as RD2_EnabledErrorMessage. @sam-knox , Please confirm if it looks good. This is a message to know the users that Enhanced RD2 is enabled already in the org.